ddtrace/_monkey.py

```diff
@@ -150,7 +151,8 @@ def on_import(hook):
         path = "%s.%s" % (prefix, module)
         try:
             imported_module = importlib.import_module(path)
-        except ImportError:
+        except Exception as e:
+            telemetry_writer.add_error(trace_telemetry.INTEGRATION_INSTALL_ERROR_CODE, str(e))
```

Comment:
this will be replaced by adding the error to `add_integration` below

Reply:
Can't we just rely on `log.error()`to add the error to the telemetry writer?